jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshSteps.java

```diff
@@ -53,24 +55,38 @@
     private int maxSoftAssertCount;
 
     private ZonedDateTime scenarioStart;
+    private ZonedDateTime logReadStartTime = null;
+    private ZonedDateTime logReadEndTime = null;
+
+    private MultiKeyMap<MultiKey, String> logCache = MultiKeyMap.multiKeyMap(new LRUMap());
```

Comment:
Please think about to use more flexible caching solution based on standard spring/java support.

Reply:
I used easiest solition to solve this problem. Cache is now cleaned before every story.
I couldn't think about any reason to keep cache for another story.
And I want to use cache in ssh report extension, so flashing it sonner wouldn't work for me.

I hope this solition is OK.